provider/internal/msp/msp_tenant_users/resource_test.go

```diff
@@ -0,0 +1,66 @@
+package msp_tenant_users_test
+
+import (
+	"fmt"
+	"github.com/CiscoDevnet/terraform-provider-cdo/internal/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"testing"
+)
+
+type Users struct {
+	Username    string
+	Role        string
+	ApiOnlyUser bool
+}
+
+var testMspManagedTenantUsersResource = struct {
+	TenantUid string
+	Users     []Users
+}{
+	Users: []Users{
+		{Username: "user1@example.com", Role: "ROLE_SUPER_ADMIN", ApiOnlyUser: false},
+		{Username: "example-api-user", Role: "ROLE_SUPER_ADMIN", ApiOnlyUser: true},
```

Comment:
don't know if you also want to test with another role

Reply:
Good call. Will change.